WireUI/Package.swift

```diff
@@ -16,7 +16,8 @@ let package = Package(
         .library(name: "WireMainNavigationUI", targets: ["WireMainNavigationUI"]),
         .library(name: "WireReusableUIComponents", targets: ["WireReusableUIComponents"]),
         .library(name: "WireSettingsUI", targets: ["WireSettingsUI"]),
-        .library(name: "WireSidebarUI", targets: ["WireSidebarUI"])
+        .library(name: "WireSidebarUI", targets: ["WireSidebarUI"]),
+        .library(name: "WireFolderPickerUI", targets: ["WireFolderPickerUI"])
```

Comment:
Do you mind ordering it alphabetically? :)

Reply:
I will fix this in https://github.com/wireapp/wire-ios/pull/2127